### PR TITLE
Analysis template implementation complete

### PR DESCRIPTION
**Pull request by jpbulloch5**
The tracking-microservice now successfully implements the analysis.yml template from the project-registry-devops repo.
Depends on: https://github.com/1053-August-Duet-Project-Registry/project-registry-devops/pull/3

### CODE REVIEW

**Review by escalonn — CHANGES_REQUESTED** [review]
cf. https://github.com/1053-August-Duet-Project-Registry/project-registry-account-microservice/pull/3#pullrequestreview-711097506

**Review by escalonn — CHANGES_REQUESTED** [review]
see https://github.com/1053-August-Duet-Project-Registry/project-registry-account-microservice/pull/3#pullrequestreview-711258014
also, the build is failing (and it should be a required check via branch policy).

File: devops/azure-pipelines.yml [hunk 1]
```diff
@@ -46,16 +48,18 @@ stages:
...
+    #     microservice: tracking-microservice
+    #     workDir: ./project-registry-tracking-microservice/
+    #     sonarKeyPrefix: $(sonarKeyPrefix)
+    #     sonarNamePrefix: $(sonarNamePrefix)
+    #     sonarOrg: $(sonarOrg)
+    #     sonarUrlPrefix: $(sonarUrlPrefix)
+    #     discordChannel: $(discordAnalysisChannel)
+    #     discordKey: $(discordAnalysisKey)
+
+    - script: echo $(Build.Repository.Uri)
```
Comment (escalonn) [inline devops/azure-pipelines.yml]: this doesn't look ready for review

File: devops/azure-pipelines.yml [hunk 2]
```diff
@@ -9,9 +10,10 @@ resources:
     ref: refs/heads/feature-templates
 
 # This pipeline will trigger on commits or pull-requests made to the branch-dev
-# or main branches.  Note that static analysis with SonarCloud is only performed
-# on "long-lived" branches, which are designated as 'main' and any branch name
-# prefixed with 'branch-'.
+# or main branches.  Note that full static analysis with SonarCloud is only 
+# performed on "long-lived" branches, which are designated as 'main' and any 
+# branch name prefixed with 'branch-'.  Minimal analysis is still performed on
+# "short-lived" branches and special analyisis is performed on pull-requests
```
Comment (escalonn) [inline devops/azure-pipelines.yml]: spelling

**Review by escalonn — CHANGES_REQUESTED** [review]

File: devops/azure-pipelines.yml (hunk 1 above)
Comment (jpbulloch5) [reply devops/azure-pipelines.yml]: Oops, having several nearly identical YAML files made this get overlooked.  Sorry.
Anyway, it is fixed now.

File: devops/azure-pipelines.yml (hunk 2 above)
Comment (jpbulloch5) [reply devops/azure-pipelines.yml]: updated grammar/spelling on comment block

**Review by escalonn — APPROVED** [review]

**Review by BeoHawk25 — APPROVED** [review]

**Pull request merged**